draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1272,6 +1272,18 @@ time period considered appropriate (e.g., randomize `iat` within the last 24
 hours and calculate `exp` accordingly) or rounded (e.g., rounded down to the
 beginning of the day).
 
+SD-JWT only conceals the value of claims that aren't revealed.
+It does not meet standard security notations for anonymous credentials. In
+particular Verifiers and Issuers can know when they have seen the same
+credential no matter what fields have been disclosed, even none of them.
+This behavior may not accord with what users intuitively expect or are
+lead to expect from user interface interactions and lead to them make choices they
+would not otherwise make. Workarounds such as issuing multiple
+credentials at once and using them only one time can help for keeping
+Verifiers from linking different presentations, but cannot work for Issuers.
```

Comment:
```suggestion
would not otherwise make. Workarounds such as batch issuance as
described above help for keeping
Verifiers from linking different presentations, but cannot work for Issuer/Verifier unlinkability.
```